src/Umbraco.Web.UI.Client/src/views/components/users/change-password.html

```diff
@@ -62,7 +58,7 @@
                     </span>
                 </umb-control-group>
 
-                <a href="" ng-click="vm.cancelChange()" ng-show="vm.showCancelBtn()" class="btn btn-small">
+                <a href="" ng-click="vm.cancelChange()" ng-show="vm.showCancelBtn()" class="btn umb-button__button btn-cancel umb-button--">
```

Comment:
@poornimanayar My above comment also goes for this line 😃 - Let me know if you need further pointers. I'll be happy to assist 👍